addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -347,7 +347,7 @@ var MassMailingFieldHtml = FieldHtml.extend({
         var $snippets_menu = $snippetsSideBar.find("#snippets_menu");
 
         for (const button of $snippets_menu.get(0).children) {
-            if (button.textContent === 'Select a template') {
+            if (button.id === 'select_template') {
```

Comment:
Maybe we could use tabIndex as a way to differenciate the buttons ?

https://github.com/odoo/odoo/blob/1ae4c26fe3979b400b3488292a7157e11d18f08d/addons/web_editor/views/snippets.xml#L9-L19

And only hide the button that has no tabIndex

Reply:
Adding tabindex would also break stable policies, because current behavior will stop working until template containing new tabindex is updated, so a module update would be required.

I take the approach of comparing against translated term.

Reply:
I don't see what you mean as there won't be any template to change. What I meant is that the only button we want to hide doesn't have a tabIndex:
https://github.com/odoo/odoo/blob/2632e9e553cc50e4273a0150bef67818b83ba8de/addons/mass_mailing/views/snippets_themes.xml#L5-L7
So we can use something like this without updating any template. 
```js
if (!button.hasAttribute('tabIndex')) {
    button.style.display = 'none';
}
```
I'm not sure if it is better than comparing against the translated term as this will not work with customization or outdated templates though :shrug:

Reply:
None of them have a name so your condition `!button.hasAttribute('name')` has no effect.

Ultimately, I think someone from @odoo/rd-framework-editor should review this as I am not knowledgeable enough.

Reply:
Done